extensions/KHR/SPV_KHR_cooperative_matrix.asciidoc

```diff
@@ -386,20 +388,26 @@ invocation when treated as a composite. +
 3.42.11 Conversion Instructions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Allow cooperative matrix types for the following conversion instructions (if
-the component types are appropriate): *OpConvertFToU*, *OpConvertFToS*,
+Allow values of cooperative matrix type for the following conversion instructions
+(if the component types are appropriate): *OpConvertFToU*, *OpConvertFToS*,
 *OpConvertSToF*, *OpConvertUToF*, *OpUConvert*, *OpSConvert*, *OpFConvert*.
-Allow the use of *OpBitcast* on cooperative matrix types whose 'Component Type'
-are integer types with the same 'Width'.
+Allow the use of *OpBitcast* on objects of cooperative matrix type whose
+'Component Type' are integer types with the same 'Width'.
 The result type and value type must have the same 'Scope', number of 'Rows',
 number of 'Columns', and 'Use'.
 
+For a given dynamic instance of these instructions, all their operands must be
+the same for all invocations in a given scope instance (where the scope is the
```

Comment:
I know some of this is preexisting, but I'm not sure how to interpret "the same" in this restriction. One could interpret it to mean they must be uniform.

Reply:
It's a carry over from the NV extension (in case you want to change the language there too). Agree, we can probably improve on this text. How about:

>All the operands to these instructions must be dynamically uniform within the 'Scope' of the cooperative matrix.